test_util/test_runner.py

```diff
@@ -51,10 +49,7 @@ def integration_test(
         'SLAVE_HOSTS=' + ','.join(agent_list),
         'PUBLIC_SLAVE_HOSTS=' + ','.join(public_agent_list),
         'DCOS_PROVIDER=' + provider,
-        'DNS_SEARCH=' + dns_search,
-        'AWS_ACCESS_KEY_ID=' + aws_access_key_id,
```

Comment:
note: Resiliency tests are just going to add these right back in anyway, so it doesnt really matter if you remove them